src/core/Akka.Remote.Tests.Performance/Transports/RemoteMessagingThroughputSpecBase.cs

```diff
@@ -21,7 +21,7 @@ namespace Akka.Remote.Tests.Performance.Transports
     public abstract class RemoteMessagingThroughputSpecBase
     {
         private const string RemoteMessageCounterName = "RemoteMessageReceived";
-        private const long RemoteMessageCount = 10000;
+        private const long RemoteMessageCount = 50000;
```

Comment:
Opted to run each individual iteration of our messaging benchmarks for a greater number of messages, but run fewer iterations (3 instead of 13.)